sunpycube/cube/cube_utils.py

```diff
@@ -467,7 +467,7 @@ def _convert_slice(item, wcs, axis, _source='cube'):
         opposite WCS convention)
     """
     if _source == 'cube':
-        wcs_ax = -2 - axis if wcs.was_augmented and not wcs.oriented \
+        wcs_ax = -2 - axis if wcs.was_augmented or not wcs.oriented \
```

Comment:
even if wcs.oriented is true /false if wcs.was_augmented is true than the wcs object has 4 dim so wcs_ax=-2-axis should be called